salt/modules/virt.py

```diff
@@ -4187,8 +4185,8 @@ def _skip_update(names):
         "disk": _skip_update(["disks", "disk_profile"]),
         "interface": _skip_update(["interfaces", "nic_profile"]),
         "graphics": _skip_update(["graphics"]),
-        "serial": _skip_update(["serial"]),
-        "console": _skip_update(["console"]),
```

Comment:
**question**: I'm guessing this is correct - just wanted to double check that `"serial"` and `"console"` should only be plural (have an S) in the list passed to `_skip_update`, correct? I see that it's not being updated in the `to_skip` dict - just the singular.

Is that expected?

Reply:
that's correct. The array contains `virt.update()` parameter names (which are plural), while the key is used in the `_diff_*_lists()` name and are singular.